@here/harp-mapview/lib/MapView.ts

```diff
@@ -3282,6 +3281,8 @@ export class MapView extends THREE.EventDispatcher {
             currentFrameEvent.setValue("renderCount.frameNumber", this.m_frameNumber);
         }
 
+        this.dispatchEvent(RENDER_EVENT);
```

Comment:
We now dispatch  the event after increasing the frameNumber. So if somebody is using MapView.frameNumber in the render event the semantics changed.
What about instead of moving the event, just take the current time again with `PerformanceTimer.now()` That would also fix the problem that we compare timestamps from two different sources (i.e. animation frame and PerformanceTimer)